### DISCUSSION TITLE
Turning off gatsby-source-wordpress live updates in development

### DISCUSSION THREAD

**Discussion by div-cowboy** [gatsbyjs/gatsby#29743] [Help]
I've handed a site off to my client, but when I try to QA it, it keeps rebuilding as the client makes updates in Wordpress. Is there a way to turn off the live updating in dev to prevent this?

**Comment by herecydev** [comment]
Assuming you're using the official wordpress plugin you could always raise the limit of the the updateInterval:

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/plugin-options.md#developnodeupdateinterval
